sdk/helper/cel/helpers.go

```diff
@@ -93,3 +70,80 @@ func AddIdentity(view logical.SystemView, req *logical.Request, data map[string]
 
 	return nil
 }
+
+func decodeJSON(value ref.Val) ref.Val {
+	raw, ok := value.Value().(string)
+	if !ok {
+		return types.Bool(false)
+	}
+	var v any
+	if err := json.Unmarshal([]byte(raw), &v); err != nil {
+		return types.Bool(false)
+	}
+	return types.DefaultTypeAdapter.NativeToValue(v)
+}
+
+func encodeJSON(value ref.Val) ref.Val {
+	native, err := value.ConvertToNative(
+		reflect.TypeOf(map[string]any{}),
+	)
+	if err != nil {
+		return types.Bool(false)
+	}
+
+	b, err := json.Marshal(native)
+	if err != nil {
+		return types.Bool(false)
+	}
+	return types.String(string(b))
+}
+
+// registerCheckValidEmailFunction registers the check_valid_email function in the CEL environment.
+func registerCheckValidEmailFunction(env *celgo.Env) (*celgo.Env, error) {
+	return env.Extend(
+		celgo.Function("check_valid_email",
+			celgo.Overload("check_valid_email_string",
+				[]*celgo.Type{celgo.StringType}, // Takes a string input
+				celgo.BoolType,                  // Returns a boolean
+				celgo.UnaryBinding(checkValidEmail),
+			),
+		),
+	)
+}
+
+// registerDecodeEncodeJSONFunctions registers decode_JSON and encode_JSON
+func registerDecodeEncodeJSONFunctions(env *celgo.Env) (*celgo.Env, error) {
+	return env.Extend(
+		celgo.Function("decode_JSON",
```

Comment:
```suggestion
		celgo.Function("decode_json",
```

I'd probably keep these lowercase :-)

Reply:
Oh yea, lemme do that :))